vendor/zlib/premake5.lua

```diff
@@ -20,3 +20,6 @@ project "zlib"
 	}
 
 	defines { "verbose=-1" }    -- Stop "bit length overflow" warning
+
+	filter "system:Mac*"
+		defines { "Z_HAVE_UNISTD_H" }
```

Comment:
Nice, I believe it should be `HAVE_UNISTD_H` to follow the [zlib configure source](https://github.com/madler/zlib/blob/cacf7f1d4e3d44d871b605da3b647f07d718623f/zconf.h.in#L434) options.

Reply:
you're right - I've updated this